app/components/LiveDock.vue

```diff
@@ -48,8 +48,12 @@
       </div>
 
       <div class="live-dock-info">
-        <a @click="showEditStreamInfo" v-if="isTwitch">Edit Stream Info</a>
-        <a @click="refreshChat">Refresh Chat</a>
+        <div class="platform-tools">
```

Comment:
Let's make this class `live-dock-platform-tools` just to match everything else in this component and ensure our class names are sufficiently unique.

Reply:
fixed 3d9cce1c30c4e8096068868317ba472e91a5cf4a